docs/design_flaws.md

```diff
@@ -735,6 +736,29 @@ ENDM
 **Fix:** Edit [home/sine.asm](https://github.com/pret/pokecrystal/blob/master/home/sine.asm) to contain a single copy of the (co)sine code in bank 0, and call it from those five sites.
 
 
+## The 6-bit caught level can only record up to level 63
+
+When a Pokémon is caught, it stores its caught data in two bytes as part of the `party_struct`. The first byte uses 2 bits for `CAUGHT_TIME` and 6 bits for `CAUGHT_LEVEL`. The second byte uses 1 bit for `CAUGHT_GENDER` and 7 bits for `CAUGHT_LOCATION`. When a Pokémon greater than level 63 is caught, the game first stores `CAUGHT_TIME` in the first two bits of the first byte, and then does an `OR`
+operation on that byte with `[wCurPartyLevel]` during execution of `SetCaughtData:`. The result is that `CAUGHT_TIME` may become corrupted, and the `CAUGHT_LEVEL` reporting incorrectly. For example:
+
+A level 70 Pokémon caught at Morning: (% means binary)
+
+`%00000000` = Morning
+
+`%01000110` = Level 70
+
+Since level 70 takes up 7 bits, when you apply an OR operation here you get:
+`%01000110` = Level 6 Pokémon caught at Day.
+The game only looks at the first two bits for Time and last 6 bits for Level due to:
+
+```asm
+DEF CAUGHT_TIME_MASK  EQU %11000000
+DEF CAUGHT_LEVEL_MASK EQU %00111111
+```
```

Comment:
```suggestion
Pokémon that are caught above level 63 (`%00111111`) overflow into `CAUGHT_TIME`. The `party_struct` for a Pokémon stores both `CAUGHT_TIME` and `CAUGHT_LEVEL` in a single byte (The first byte of two in `CAUGHT_DATA`). The game uses `CAUGHT_TIME_MASK` (`%11000000`) and `CAUGHT_LEVEL_MASK` (`%00111111`) when reading from the byte. When a Pokémon is caught it first stores `CAUGHT_TIME` into the byte and rotates them right twice to occupy bits 6 & 7. Then it performs an `OR` operation on that byte with the value in `[wCurPartyLevel]`. This means a level 70 Pokémon (`%01000110`) caught at Morning (`%00000000` or `$00`) is stored as (`%01000110`), and read as a Level 6 Pokémon (`%00000110`) Caught at Day (`%01000000` or `%01`).
```

I think I like this wording better. Chloe please **do not** apply this suggestion right away, I would like Rangi to look and determine which looks better.

Reply:
In my opinion, the wording you suggest gives a more detailed description of the problem that would make more sense to the average user.
Still would like to see what Rangi has to say on this when she has time